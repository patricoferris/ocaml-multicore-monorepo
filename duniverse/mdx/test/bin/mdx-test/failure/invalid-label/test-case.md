`toto` is not a valid label:

```sh toto,dir=xxx
$ xxx
```
