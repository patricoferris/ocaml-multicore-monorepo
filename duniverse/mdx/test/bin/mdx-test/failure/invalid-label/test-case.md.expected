[mdx] Fatal error: File "test-case.md", line 3: invalid code block: `toto` is not a valid label.
